app/popups/ServerPasswordPopup.js

```diff
@@ -51,6 +79,10 @@ class ServerPasswordPopup extends Component
 
         if (this.refs.password)
             this.refs.password.focus();
+
+        this.setState({
+            isCapsLockOn: false
+        })

```

Comment:
Semicolons please :)

Also, I don't think this state reset won't be needed if the other comments are addressed.

Reply:
Correct, constructor takes care of it.